r/tests/testthat/helper-skip.R

```diff
@@ -21,7 +21,10 @@ build_features <- c(
   uncompressed = TRUE
 )
 
+force_tests <- identical(tolower(Sys.getenv("ARROW_R_FORCE_TESTS")), "true")
+
 skip_if_not_available <- function(feature) {
+  if (force_tests) return()
```

Comment:
This is a minor style thing, but if there's an early return, we should make sure that goes on a separate line:

```suggestion
  if (force_tests) {
    return()
  }
```